### ISSUE TITLE
Automatic mode on overwrites open in delete history setting (ctrl+click)

### ISSUE THREAD

**Issue by RaitaroH** [stoically/temporary-containers#413]
- Temporary Containers Version: 1.8
- Firefox Version: waterfox 2020.01 (64-bit)

## Actual behaviour
Considering the following settings. Automode overwrites any attempt to open a delete history tab manually unless I use the keyboard shortcut. Alternatively if I make all tabs to be delete history than this will work, but that is not my intent.
![image](https://user-images.githubusercontent.com/19552720/82724737-f8437800-9ce0-11ea-9a44-69e96224d612.png)

## Steps to reproduce
1. open a delete history tab using shortcut
2. search test on duckduckgo
3. now we are in a delete history tab with duckduckgo
4. ctrl+click on speedtest.net 
5. speedtest opens in normal tmp containers, every time

## Expected behaviour
The setting should work as described and open a delete history tab.

Is it possible to have automatic mode on so that sites open in temporary containers, but, have some sort of setting, so that certain domains always open in delete history mode? This should work even if the url is opened from non-temp containar tab, delete history container tab, permanent container tab, history / bookmarks. Pretty much if the link is part of a domain, always open in delete history.

**Comment by stoically** [comment]
You're right, that seems to be broken indeed. Thanks for letting me know.

> Pretty much if the link is part of a domain, always open in delete history.

Interesting idea. I've added that to the list of #397 - that'd be good fit there and allow removing that confusing history preferences screen.
